usng.js

```diff
@@ -46,6 +46,41 @@
     }
 }(this, function (root, usngs) {
 
+    function eatanhe(x) {
+        return this.es * Math.atanh(this.es * x)
+    }
+
+    function taupf(tauValue) {
+        const tau1 = Math.hypot(1.0, tauValue)
+        const sig = Math.sinh(eatanhe.call(this, tauValue / tau1))
+        return Math.hypot(1.0, sig) * tauValue - sig * tau1
+    }
+
+    function tauf(taupValue) {
+        const e2m = 1 - Math.pow(this.es, 2)
+            // To lowest order in e^2, taup = (1 - e^2) * tau = _e2m * tau; so use
+            // tau = taup/_e2m as a starting guess.  (This starting guess is the
+            // geocentric latitude which, to first order in the flattening, is equal
+            // to the conformal latitude.)  Only 1 iteration is needed for |lat| <
+            // 3.35 deg, otherwise 2 iterations are needed.  If, instead, tau = taup
+            // is used the mean number of iterations increases to 1.99 (2 iterations
+            // are needed except near tau = 0).
```

Comment:
✏️ This should be a comment using `/* */`

Reply:
That would be inconsistent with the existing comments.